### ISSUE TITLE
lief.PE.ResourcesManager is segfaulting

### ISSUE THREAD

**Issue by Rafiot** [lief-project/LIEF#50]
PEs without `ResourcesManager` will segfault:
```python
In [1]: import lief                                                                                                                                 
                                                                                                                                                    
In [2]: a = lief.parse("/home/raphael/.viper/projects/troopers17/vt_samples/2122/1a09ce8a9210d2530d6ce1d59bfae2ac617ac89558cdcdcac15392d176e70c8d") 
                                                                                                                                                    
In [3]: a.resources_manager                                                                                                                         
Out[3]: <lief.PE.ResourcesManager at 0x7f4bf090c5a0>                                                                                                
                                                                                                                                                    
In [4]: a.resources_manager.<TAB>Segmentation fault (core dumped)                                                                                        
```

Printing/to_json also fails

**Comment by romainthomas** [comment]
I think the segfault has been fixed with #47

**Comment by Rafiot** [comment]
woops, right, sorry. I'm using the latest release and didn't search in the closed issues ;)
